chicken-sloths/bangazon-cli-sprint2
# Joe active cust c

# Description
This PR makes the "Select Active Customer" actually select a customer. 
It displays all customers in the database, in a list with just their id number and full name. 
This PR also makes getActiveCustomer give us back just the id of the active customer rather than the activeCustomer object. After selecting an active customer, the user will be directed back to the main menu, AND it DISPLAYS THE ID OF THE ACTIVE USER ON THE MAIN MENU TALM BOUT SAYY WHHAAAAATTTTT

## Related Ticket(s)
Fixes #2 

## Problem to Solve
Gives user ability to select the active user as a static value.

## Proposed Changes
I TESTED TO SEE WHAT WOULD HAPPEN IF I CHOSE AN ID OF A NON-EXISTING CUSTOMER (LIKE 1000), AND IT STILL LETS YOU SELECT THAT, AND IT WILL DISPLAY IT ON THE MAIN MENU. WE NEED SOME SORT OF CHECK TO MAKE SURE THE ID CHOSEN ACTUALLY EXISTS FOR A CUSTOMER

## Expected Behavior
Be able to select an active user, and see that users id displayed on the main menu

## Steps to Test Solution

1. Run `npm start`
2. Select `2` from the main menu
3. See that the customers are displayed in the console,
4. choose an existing customer id
5. You should be return to the main menu and see that user id displayed right above the menu, in blue 

## Testing


## Documentation
